lib/turbo/topup/views/topup_modal.dart

```diff
@@ -54,6 +55,8 @@ void showTurboModal(BuildContext context, {Function()? onSuccess}) {
 
   initializeStripe(context.read<ConfigService>().config);
 
+  context.read<ActivityTracker>().setToppingUp(true);
```

Comment:
Will change if you replace the method approach with the setter approach
```suggestion
  context.read<ActivityTracker>().setToppingUp = true;
```